EnglishFilter/sections/foreign.txt

```diff
@@ -4698,6 +4698,8 @@ grandhotelcalciomercato.com#$#.marginTopNavbar { margin-top: auto !important; }
 grandhotelcalciomercato.com#%#//scriptlet('set-constant', 'canRunAds', 'true')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/71176
 veneziatoday.it#@#.afs_ads
+! https://github.com/AdguardTeam/AdguardFilters/issues/71313
+youmath.it#%#//scriptlet('abort-on-property-write', 'is_adblock')
```

Comment:
What about platforms without JS?

Reply:
@Alex-302, sadly, no one was able to reproduce it, and i got this rule with the help of the user.

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/105981166-d3d94000-60a6-11eb-87d4-af6e62fef595.png)
</details><br/>

Reply:
Scriptlets cannot be applied on iOS.